e2e/test/scenarios/collections/instance-analytics.cy.spec.js

```diff
@@ -142,5 +144,86 @@ describeEE("scenarios > Instance Analytics Collection", () => {
         cy.contains("Move").should("not.exist");
       });
     });
+
+    it.skip("should not allow editing analytics content (#36228)", () => {
```

Comment:
Nit: only for consistency (although we can easily stop prefixing issue numbers with the repo name.

```suggestion
    it.skip("should not allow editing analytics content (metabase#36228)", () => {
```